cmd/podman/libpodruntime/runtime.go

```diff
@@ -41,6 +41,9 @@ func GetRuntimeWithStorageOpts(c *cli.Context, storageOpts *storage.StoreOptions
 	if c.GlobalIsSet("conmon") {
 		options = append(options, libpod.WithConmonPath(c.GlobalString("conmon")))
 	}
+	if c.GlobalIsSet("tmpdir") {
+		options = append(options, libpod.WithTmpDir(c.GlobalString("tmpdir")))
```

Comment:
This seems unrelated to rootless-ness?

Reply:
true, I've added it for local testing and I thought it could be in general useful to expose this setting

Reply:
> ... I thought it could be in general useful to expose this setting

So make it a separe PR?  It will probably settle faster that way withough the rest of the rootless stuff.